_Tutorials/16-javascript-es6/16.8-filter.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Array Functions: filter()"
+video_number: 8
+date: 2018-02-22
+video_id: qmnH5MT_luk
+
+videos:
+  - title: "The Modulo Operator with Golan Levin"
+    url: "/GuestTutorials/6-golan-levin-modulo-operator"
+  - title: "Regular Expressions"
+    url: "https://www.youtube.com/watch?v=7DG3kCDx53c"
```

Comment:
That playlist already exists here: http://thecodingtrain.com/Courses/programming-with-text/2-regular-expressions/
Please use the internal link instead.